Modules/Module_Processing/Module_Population/xASL_stat_GetROIstatistics.m

```diff
@@ -762,6 +782,17 @@
                 elseif xASL_stat_SumNan(pvSecondary(:)) == 0
                     fprintf('%s\n', ['* Empty pv' pvSecondaryName ' for ' x.SUBJECTS{iSubject} '_ASL_' xASL_num2str(iSess) ', ROI ' xASL_num2str(iROI) ':' namesROIuse{iROI}]);
                 else                    
+                    % Check if the ROI size is large enough
+                    imMask = CurrentMaskNotVascular;
+                    imMask = (imMask>0) & isfinite(DataIm);
+                    imMask = imMask & (DataIm~=0); % Exclude zero values as well
+
+                    if sum(imMask)<MinVoxels
+                        fprintf('\n');
+                        warning('%s\n', [x.S.TissueMaskingLocal ' ' namesROIuse{iROI} ' only contains ' xASL_num2str(sum(imMask)) ' voxels, so this ROI will be skipped in sCoV calculations']);

```

Comment:
same as above

Reply:
Done in 46c421bcd1aed8156cbe146a8cf708930de3d337